### PR TITLE
support to exclude specific tags or edges when migrate space

### PR DESCRIPTION
**Pull request by Nicole00**

### CODE REVIEW

File: example/src/main/scala/com/vesoft/nebula/examples/connector/Nebula2Nebula.scala [hunk 1]
```diff
@@ -55,18 +55,38 @@ object Nebula2Nebula {
...
+    writeParallelOption.setRequired(true)
+    val timeoutOption = new Option("timeout", "timeout", true, "timeout for java client");
+    timeoutOption.setRequired(true)
+    val userOption = new Option("u", "user", true, "user")
+    userOption.setRequired(true)
+    val passwdOption = new Option("passwd", "password", true, "password")
+    passwdOption.setRequired(true)
+
+    // filter out some tags /edges
+    val missTagsOption =
```
Comment (wey-gu) [inline example/src/main/scala/com/vesoft/nebula/examples/connector/Nebula2Nebula.scala:86]:
```suggestion
    val excludeTagsOption =
```

Comment (wey-gu) [reply example/src/main/scala/com/vesoft/nebula/examples/connector/Nebula2Nebula.scala:86]: great work!

chore: small wording suggestion

Comment (Nicole00) [reply example/src/main/scala/com/vesoft/nebula/examples/connector/Nebula2Nebula.scala]: great!

**Comment by Nicole00** [comment]
please help to review this pr again, thanks @wey-gu

**Review by wey-gu — APPROVED** [review]
great job, thanks!!

**Pull request merged**